app/Http/Controllers/Admin/RoleController.php

```diff
@@ -133,6 +146,12 @@ public function destroy(Role $role)
     {
         $this->checkPermission(self::WRITE_PERMISSIONS);
 
+        if ($role->hasAllPermissions() && $role->count() === 1) {
```

Comment:
![image](https://user-images.githubusercontent.com/45005889/201467983-6432c6c2-11bf-4056-a571-fb2dc4399af5.png)
Well try and stop me XD, I was testing if I would be unable to delete an extra admin role. this would be rather annoying if you can't delete admin roles as an admin. but I was able to both delete my extra role and my own admin role locking me out. so I see where this pr is coming from. but I think it doesn't work.